### DISCUSSION TITLE
Windows Powershell questions help

### DISCUSSION THREAD

**Discussion by farmingtonschia** [Chia-Network/chia-blockchain#2460] [Q&A]
I tried to follow the wiki https://github.com/Chia-Network/chia-blockchain/wiki/Windows-Tips-and-Tricks#1-parallel-plotting-using-powershell I have not been successful 

I have windows powershell 5.1 and I tried to run as the user and not the administrator - would these be cause of my following difficulties I am having?

1. I tried setting up parallel plotting using the examples given in the wiki and kept getting error - how the argument list of structured. If there is a screenshot of one that worked that would be great
2. I tried to but could not  locate the powershell > properties>options to  disable the quickedit , 
3. if I delete the wallet - how to get reinstate it and what happens to what was in the wallet

**Comment by dorkmo** [comment]
3. you need to add the 24 words again with `chia keys add` https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference

**Reply by farmingtonschia** [reply]
Thank you that is very helpful
